app/build.gradle

```diff
@@ -1,6 +1,8 @@
 plugins {
     id 'com.android.application'
     id 'org.jetbrains.kotlin.android'
+    // Room DB
```

Comment:
I know that kotlin-kapt is a plug-in related to annotation! So I think it's better to delete this line.

Reply:
Oh, I didn't realized it. Thanks for feedback!! I applied it!